gcloud-java-core/src/main/java/com/google/gcloud/ServiceOptions.java

```diff
@@ -53,6 +53,7 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+@SuppressWarnings("rawtypes")
```

Comment:
I think it is a bad idea to apply this to a whole class, especially one as big as this one. The fact that you feel the need says to me that there is something wrong with the class. Do we really need all the type parameters?